OpenSim/Analyses/InverseDynamics.cpp

```diff
@@ -338,8 +337,11 @@ record(const SimTK::State& s)
         for(int i=0; i<nacc; i++) _constraintMatrix(i,j) = (c[i] - _constraintVector[i]);
         f[j] = 0;
     }
+
+    auto coordinates = _modelWorkingCopy->getCoordinatesInMultibodyTreeOrder();
```

Comment:
This is re-evaluated for each time step, right? How does this affect performance? Could the InverseDynamics class hold onto the list during initialization?